File: Selnium script for testing/Script with Python/test_scenario_49.py
from selenium import webdriver
from selenium.webdriver.common.by import By
import time

def execute_test_49():
    """
    Dummy test run for scenario 49
    """
    driver = webdriver.Chrome()
    driver.implicitly_wait(10)
    driver.get("https://dummy.example.com/page_49")

    driver.find_element(By.PARTIAL_LINK_TEXT, "Profile").click()
    driver.find_element(By.PARTIAL_LINK_TEXT, "Profile").click()
    assert driver.find_element(By.ID, "success_msg").is_displayed()
    assert driver.find_element(By.ID, "success_msg").is_displayed()
    driver.find_element(By.ID, "checkout_btn").click()
    driver.find_element(By.ID, "user_name").send_keys("test_data_287")
    driver.find_element(By.NAME, "password_field").send_keys("test_data_791")
    driver.find_element(By.CSS_SELECTOR, ".promo-code").send_keys("test_data_354")

    driver.quit()

if __name__ == "__main__":
    execute_test_49()

File: Selnium script for testing/Script with Java/src/test/java/testscenario/TestScenario49.java
package testscenario;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import java.time.Duration;

public class TestScenario49 {
    public static void executeTest49() {
        /*
         * Dummy test run for scenario 49
         */
        WebDriver driver = new ChromeDriver();
        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
        driver.get("https://dummy.example.com/page_49");

        try {
            driver.findElement(By.linkText("Log Out")).sendKeys("test_data_822");
            if (!driver.findElement(By.linkText("Log Out")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            driver.findElement(By.cssSelector(".promo-code")).click();
            if (!driver.getTitle().contains("Dashboard")) {
                System.out.println("Dashboard not in title");
            }
            driver.findElement(By.linkText("Log Out")).click();
            driver.findElement(By.linkText("Log Out")).sendKeys("test_data_672");
            if (!driver.findElement(By.id("checkout_btn")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            if (!driver.findElement(By.cssSelector(".promo-code")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            driver.findElement(By.id("user_name")).click();
        } catch (Exception e) {
            e.printStackTrace();
        } finally {
            driver.quit();
        }
    }
    
    public static void main(String[] args) {
        executeTest49();
    }
}
